content/en/cloudcraft/advanced/auto-layout-via-api.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Automate Snapshots of Cloud Accounts via the Cloudcraft API
+kind: guide
+---
+
+## Overview
+
+Cloudcraft's **Auto Layout** feature, accessible through the web application, is a powerful tool for automatically generating diagrams of your AWS environment. This functionality can significantly streamline documentation processes and facilitate the onboarding of new team members.
+
+This guide provides a step-by-step approach to utilizing this feature via common command-line utilities and the Cloudcraft developer API.
+
+<div class="alert alert-info">The ability to add and scan AWS accounts, as well as to use Cloudcraft's developer API, is only available to Pro subscribers. Check out <a href="https://www.cloudcraft.co/pricing">Cloudcraft's pricing page</a> for more information.</div>
+
+## Prerequisites
+
+- An active [Cloudcraft Pro subscription][1].
+- An API key with read-write permissions.
+- Access to a Unix-like environment (Linux, macOS, or Windows Subsystem for Linux).
+- Familiarity with command-line operations.
+- Basic knowledge of API usage.
+
+## Take a snapshot of the account
+
+Start by creating a snapshot of your AWS or Azure account using the [Snapshot AWS account][2] or [Snapshot Azure account][3] endpoints. This process mirrors the functionality of the **Scan Now** button in the Cloudcraft UI and outputs the snapshot in JSON format.
+
+Execute the following command in your terminal:
+
+**For AWS:**
```

Comment:
Instead of calling out separately for AWS and Azure , you can also make that a parameter like PROVIDER where provider value is `aws` or `azure`

Reply:
Oh, great idea! Doing that now.

Reply:
Fixed with [685aacc](https://github.com/DataDog/documentation/pull/22262/commits/685aacc3b5f5d89410ea3802276ae65ce1f19b4a).